infrastructure/iac/template/index.html

```diff
@@ -82,7 +83,17 @@ <h2>References</h2>
       {{end}}
     </div>
   </section>
+  <!-- Ignore In Line Actions -->
+  {{if not .Issue.IsIgnored}}
```

Comment:
comment: strictly speaking this is not necessary, but nice for later, when ignores are provided from API/Findings model.